### PR TITLE
DOC: add a 1.26.1 release notes section for BLAS/LAPACK build changes

### PR DESCRIPTION
**Pull request by rgommers**

### CODE REVIEW

**Comment by charris** [comment]
Thanks Ralf.

File: doc/source/release/1.26.1-notes.rst [hunk 1]
```diff
@@ -3,6 +3,58 @@
...
+  for in order, overriding the default search order.
+- ``-Duse-ilp64``: if set to ``true``, use ILP64 (64-bit integer) BLAS and
+  LAPACK. Note that with this release, ILP64 support has been extended to
+  include MKL and FlexiBLAS. OpenBLAS and Accelerate were supported in previous
+  releases.
+- ``-Dallow-noblas``: if set to ``true``, allow NumPy to build with its
+  internal (very slow) fallback routines instead of linking against an external
+  BLAS/LAPACK library. *The default for this flag may be changed to ``false``
+  in a future 1.26.x release, however for 1.26.1 we'd prefer to keep it as
+  ``true`` because if failures to detect an installed library are happening,
```
Comment (rgommers) [inline doc/source/release/1.26.1-notes.rst:51]: Oops, spotted a mistake here. The first word here should be `false`, and two lines up it should be `true`. @charris would you prefer me to open a new PR to fix it, or can you take it along in your release notes update that is in the pipeline I think?

Comment (charris) [reply doc/source/release/1.26.1-notes.rst:51]: I'll do it now before I forget :)

Comment (charris) [reply doc/source/release/1.26.1-notes.rst:51]: Done.

**Pull request merged**